TodoListManagerTest/src/test/java/PerformanceTest/ProjectsPerformanceTest.java

```diff
@@ -0,0 +1,342 @@
+        runtime.gc();
+    }
+
+    @Test
+    public void createProjectsJsonPut(){
+        List<Long> timeTable = new ArrayList<Long>();
+        for(int i=0;i<1000;i++){
+            JSONObject body = new JSONObject();
+            String newTitle = "New project " + i;
+            Boolean newCompleted = false;
+            Boolean newActive = false;
+            String newDescription = "New Description";
+            body.put("title",newTitle);
+            body.put("completed",newCompleted);
+            body.put("active",newActive);
+            body.put("description",newDescription);
+            long start = System.nanoTime();
+            Response response = RestAssured.given().contentType(ContentType.JSON).body(body.toString()).put(pUrl);
+            long end = System.nanoTime();
+            long time = end-start;
+            timeTable.add(time);
+        }
+        System.out.println(timeTable);
+        Runtime runtime = Runtime.getRuntime();
+        runtime.gc();
+    }
+
+    /*
+    @Test
+    public void createProjectsXmlPost(){
+        List<Long> timeTable = new ArrayList<Long>();
+        for(int i=0;i<1000;i++){
+            JSONObject body = new JSONObject();
+            String newTitle = "New project " + i;
+            Boolean newCompleted = false;
+            Boolean newActive = false;
+            String newDescription = "New Description";
+            body.put("title",newTitle);
+            body.put("completed",newCompleted);
+            body.put("active",newActive);
+            body.put("description",newDescription);
+            String xmlBody = XML.toString(body);
+            long start = System.nanoTime();
+            Response response = RestAssured.given().contentType(ContentType.XML).body(xmlBody).post(pUrl);
+            long end = System.nanoTime();
+            long time = end-start;
+            timeTable.add(time);
+        }
+        System.out.println(timeTable);
+        Runtime runtime = Runtime.getRuntime();
+        runtime.gc();
+    }
+
+     */
+
+    @Test
+    public void createProjectsXmlPut(){
+        List<Long> timeTable = new ArrayList<Long>();
+        for(int i=0;i<1000;i++){
+            JSONObject body = new JSONObject();
+            String newTitle = "New project " + i;
+            Boolean newCompleted = false;
+            Boolean newActive = false;
+            String newDescription = "New Description";
+            body.put("title",newTitle);
+            body.put("completed",newCompleted);
+            body.put("active",newActive);
+            body.put("description",newDescription);
+            String xmlBody = XML.toString(body);
+            long start = System.nanoTime();
+            Response response = RestAssured.given().contentType(ContentType.XML).body(xmlBody).put(pUrl);
+            long end = System.nanoTime();
+            long time = end-start;
+            timeTable.add(time);
+        }
+        System.out.println(timeTable);
```

Comment:
if you are adding to a table and then printing all at the end. Wouldn't it be more optimal to print it each time when needed (e.g i == 0, i == 1000...). This way you won't need to have a list